client/src/components/RequestForm/RequestForm.tsx

```diff
@@ -1,24 +1,54 @@
 import { DateTimePicker } from '@mui/lab';
 import { Button, Card, Grid, Rating, TextField, Typography } from '@mui/material';
 import { FormEvent, useState } from 'react';
+import { useAuth } from '../../context/useAuthContext';
 import { useSnackBar } from '../../context/useSnackbarContext';
+import { createBooking } from '../../helpers/APICalls/requests';
 import { Profile } from '../../interface/Profile';
 
 interface PropTypes {
   profile: Profile;
 }
+
 const MILLISECONDS_PER_HOUR = 3.6e6;
+
+const placeholderAverageRating = 3;
+
 const RequestForm = ({ profile }: PropTypes) => {
   const hourFromNowTimestamp = Date.now() + MILLISECONDS_PER_HOUR;
   const [dropoff, setDropoff] = useState<Date | null>(new Date());
   const [pickup, setPickup] = useState<Date | null>(new Date(hourFromNowTimestamp));
+  const { loggedInUser } = useAuth();
   const { updateSnackBarMessage } = useSnackBar();
+  const [isButtonDisabled, setIsButtonDisabled] = useState(false);
 
-  const handleSubmit = (e: FormEvent) => {
+  const handleSubmit = async (e: FormEvent) => {
     e.preventDefault();
-    updateSnackBarMessage('Request sent');
+    if (!dropoff || !pickup) {
+      updateSnackBarMessage('Dropoff and pickup must be valid');
+      return;
+    }
+    const reqBody = {
+      sitter: profile.userId,
+      start: dropoff,
+      end: pickup,
+    };
+    try {
+      setIsButtonDisabled(true);
+      const { success } = await createBooking(reqBody);
+      if (success) {
+        updateSnackBarMessage('Request sent');
+      } else {
+        updateSnackBarMessage('Error sending request');
+      }
+    } catch (e) {
+      updateSnackBarMessage('Error sending request');
+    } finally {
+      setIsButtonDisabled(false);
+    }
   };
 
+  profile.averageRating = placeholderAverageRating;
```

Comment:
Similarly, here.

Reply:
I think that the intention for average rating is to come from its own api route and not the profile. After talking with the team it seems like the best thing to do is to hard code the rating into the component until the rating feature is implemented rather than attaching the placeholder rating to the profile